code/game/objects/effects/effect_system/effects_chem_smoke.dm

```diff
@@ -100,18 +100,19 @@
 /datum/effect_system/smoke_spread/chem/start(effect_range = 2)
 	eff_colour = mix_color_from_reagents(chemholder.reagents.reagent_list)
 	eff_range = effect_range
-	START_PROCESSING(SSprocessing, src)
+	START_PROCESSING(SSfastprocess, src)
 
 /datum/effect_system/smoke_spread/chem/process()
 	process_count++
-	if(eff_range < 3)
-		new /obj/effect/particle_effect/chem_smoke/small(location, eff_colour)
-	else
-		new /obj/effect/particle_effect/chem_smoke(location, eff_colour)
-
-	INVOKE_ASYNC(src, PROC_REF(SmokeEm), eff_range)
+	for(var/i = 0, i < (2 * rand(2, 6)), i++) //Every 0.2 seconds, create 4-12 smoke particles. This keeps it consitant with the 2-6 every 0.1 seconds before
+		if(eff_range < 3)
+			new /obj/effect/particle_effect/chem_smoke/small(location, eff_colour)
+		else
+			new /obj/effect/particle_effect/chem_smoke(location, eff_colour)
+	if(process_count % 5 == 0) //Every 5 ticks, or 1 second.
```

Comment:
Isn't this 10 ticks?
```suggestion
	if(process_count % 5 == 0) //Every 10 ticks, or 1 second.
```

Reply:
10 ticks, 5 ssfastprocesses, will adjust